lib/gooddata/models/project_creator.rb

```diff
@@ -13,6 +13,27 @@ module GoodData
   module Model
     class ProjectCreator
       class << self
+        def migrate_async(opts = {})
+          opts = { client: GoodData.connection, execute_ca_scripts: true }.merge(opts)
+          client = opts[:client]
+          fail ArgumentError, 'No :client specified' if client.nil?
+
+          spec = opts[:spec] || fail('You need to provide spec for migration')
+          bp = ProjectBlueprint.new(spec)
+          fail GoodData::ValidationError, "Blueprint is invalid #{bp.validate.inspect}" unless bp.valid?
+          spec = bp.to_hash
+
+          project = opts[:project] || client.create_project(opts.merge(:title => opts[:title] || spec[:title], :client => client, :environment => opts[:environment]))
```

Comment:
Identical code found in project_creator.rb:26, project_creator.rb:47 (mass = 96)